components/automate-ui/src/app/pages/automate-settings/automate-settings.component.scss

```diff
@@ -24,7 +24,7 @@ h2 {
     display: flex;
 
     input {
-      width: 60px;
+      width: 64px;
```

Comment:
bonus: noticed these inputs were not wide enough for 2 digits so increased their width by 4px